app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -195,6 +196,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
   fun compute(
     ephemeralState: EphemeralState,
     gcsEntityId: String,
+    rawUserAnswer: RawUserAnswer?,
```

Comment:
This should be passed in via the assembler's constructor & factory rather than compute() since it doesn't get recomputed each time and only exist once for the lifetime of the assembler. That should simplify the data pipeline a bit.